rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -292,11 +292,13 @@ def _check_OOV_present(self, all_tokens: List[List[Text]]) -> None:
                     return
 
         if any(text for tokens in all_tokens for text in tokens):
+            training_data_type = "NLU" if attribute == TEXT else "response"
```

Comment:
should we maybe say `ResponseSelector` instead of `response`? because `response` also refers to Core responses

Reply:
make sense